OpenOversight/app/main/views.py

```diff
@@ -1094,11 +1094,8 @@ def upload(department_id, officer_id=None):
         if officer_id:
             image.is_tagged = True
             image.contains_cops = True
-            cropped_image = crop_image(image, department_id=department_id)
-            cropped_image.contains_cops = True
-            cropped_image.is_tagged = True
             face = Face(officer_id=officer_id,
-                        img_id=cropped_image.id,
+                        img_id=image.id,
```

Comment:
Do you mind adding a comment here saying that we're setting the "cropped" image to the original image since photos are presumed to be cropped already?

Reply:
Yup!